components/light/fastled.rst

```diff
@@ -26,11 +26,122 @@ FastLED Light
 
     For addressable lights, you can use :doc:`esp32_rmt_led_strip`.
 
+.. _fastled:
+
+FastLED
+-------
+
+The ``fastled`` light platform allows you to create RGB lights in ESPHome
+for a :ref:`number of supported chipsets <fastled-chipsets>`.
+
+Both clockless and SPI chips are supported, with the former only requiring a
+single data wire to connect and the latter also needing a wire for the clock
+signal.
+
```

Comment:
```suggestion

.. warning::

    The ``fastled_spi`` and the ``fastled_clocless`` light platform are now deprecated.  
    Use the new, unified ``fastled`` light platform, instead.
```
My suggestion is to move the warning from the bottom of this document to here and remove the section about the ``fastled_spi`` and the ``fastled_clocless``.